envoy/config/rbac/v2alpha/rbac.proto

```diff
@@ -0,0 +1,155 @@
+syntax = "proto3";
+
+import "validate/validate.proto";
+
+package envoy.config.rbac.v2alpha;
+
+// RBAC (Role Based Access Control) defines ServiceRole and ServiceRoleBinding
+// objects.
+//
+// A ServiceRole specification includes a list of rules (permissions). Each rule has
+// the following standard fields:
+// * services: a list of services.
+// * verbs: HTTP verbs. In the case of gRPC, this field is ignored because the value is always
+// "POST".
+// * paths: HTTP paths or gRPC methods. Note that gRPC methods should be
+//   presented in the form of "packageName.serviceName/methodName".
+//
+// The fields used in ServiceRole definition map to attributes defined in AttributeContext
+// (//envoy/service/auth/v2/attribute_context.proto) as follows:
+//   service: destination.service
+//   method: request.http.method
+//   path: request.http.path
+//
+//  In addition, users can specify "constraints" on any attributes defined in AttributeContext
+//  (//envoy/service/auth/v2/attribute_context.proto).
```

Comment:
These links to proto files would benefit from being `:api:` references (here and elsewhere).

Reply:
Done.